modules/op-running-as-root-user-namespace.adoc

```diff
@@ -0,0 +1,46 @@
+[id='op-running-as-root-user-namespace_{context}']
```

Comment:
Every file must begin with its `:_content-type:` Please refer to the mod docs standard for more information.